src/components/ProFullCourt/index.tsx

```diff
@@ -14,6 +14,9 @@ import { useStoreSelector } from "@/store/hooks";
 import DashedLine from "../BasketballCourt/DashedLine";
 import BorderDimension from "../BasketballCourt/BorderDimensionLine";
 import { debouncedCalculation } from "@/utils/tileNumberCalculator";
+import { useDispatch } from "react-redux";
+import { changeTileQuantity } from "@/store/reducer/tileSlice";
+import { clearTimeout, setTimeout } from "timers";

```

Comment:
If I'm not wrong. This timers lib is from Node. When you use it on frontend, especially using Nextjs, it will work. But I don't see any difference between timers and web API clearTimeout/setTimeout.
Or if you have any reason using this lib?

Reply:
Thanks! Imported timer lib by mistake. Removed.